tests/__init__.py

```diff
@@ -35,8 +35,8 @@
 if not os.environ.get('GOOGLE_APPLICATION_CREDENTIALS'):
     os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = get_gcp_credentials_file().name
 with open(os.environ['GOOGLE_APPLICATION_CREDENTIALS'], "r") as fh:
-    os.environ['ADMIN_USER_EMAILS'] = json.loads(fh.read())['client_email']
-bundles.ADMIN_USER_EMAILS = set(os.environ['ADMIN_USER_EMAILS'].split(","))
+    gcp_service_email = json.loads(fh.read())['client_email']
+bundles.ADMIN_USER_EMAILS = set(["", gcp_service_email])
```

Comment:
Why is an empty string included in the set?

Reply:
@xbrianh it was per your previous comment, i shall remove the blank string in the next commit.